go/pkg/db/queries/key_list_live_by_api_id.sql

```diff
@@ -0,0 +1,95 @@
+-- name: ListLiveKeysByApiID :many
+SELECT
+    k.*,
+    sqlc.embed(a),
+    sqlc.embed(ka),
+    sqlc.embed(ws),
```

Comment:
when querying the api seperatly maybe even adding the cache to it already we can remove this.

Reply:
I didn't understand. Can you elaborate?

Reply:
If we query the API / keyauth data in a seperate query, we don't need to load in with the keys anymore

Reply:
and what about keyauth and workspace? do we need to select that multiple times?